lineapy/system_tracing/_function_calls_to_object_side_effects.py

```diff
@@ -73,7 +73,10 @@ def pointer_to_objects(
     elif isinstance(pointer, AllPositionalArgs):
         yield from function_call.args
     elif isinstance(pointer, BoundSelfOfFunction):
-        yield function_call.fn.__self__  # type: ignore
+        if hasattr(function_call.fn, "__self__"):
+            yield function_call.fn.__self__  # type: ignore
+        else:
+            yield from []
```

Comment:
not sure why all these are coming up. probably need to rebase?

Reply:
I messed up my rebase and inserted my commit before LIN-417. I need to figure out how to fix it